components/tree-table/src/tree-table-toggle-cell.ts

```diff
@@ -61,6 +63,45 @@ export class DtTreeTableToggleCell<T> extends DtCell
   /** Aria reference to a label describing the toggle button. */
   @Input('aria-labelledby') ariaLabelledBy: string;
 
+  /** Event emitted when the cell's expandable state changes. */
+  @Output()
+  readonly expandChange: Observable<
+    boolean
+  > = this._treeControl.expansionModel.changed.pipe(
+    filter(
+      (changed: SelectionChange<T>) =>
+        (changed.added.includes(this._rowData) ||
+          changed.removed.includes(this._rowData)) &&
+        this._treeControl.isExpandable(this._rowData),
+    ),
+    map((changed: SelectionChange<T>) => changed.added.includes(this._rowData)),
+  );
+
+  @Input()
+  set expanded(value: boolean) {
+    const shouldExpand = coerceBooleanProperty(value);
+
+    // Only update expanded state if it actually changed.
+    if (this._isExpanded !== shouldExpand) {
+      if (shouldExpand) {
+        this._expand();
+      } else {
+        this._collapse();
+      }
+    }
+  }
+
+  /** @internal Event emitted when the panel is expanded. */
+  @Output('expanded')
+  readonly _treeExpanded: Observable<boolean> = this.expandChange.pipe(
```

Comment:
Please remove @internal and you only have one expanded so there is no need for renaming the variable.

```suggestion
  @Output()
  readonly expanded: Observable<boolean> = this.expandChange.pipe(
```